src/org/intellij/grammar/generator/ParserGenerator.java

```diff
@@ -769,7 +779,7 @@ else if (javaType != Java.INTERFACE && i == 1) {
         sb.append(" implements ").append(shortener.shorten(aSuper));
       }
       else {
-        sb.append(", ").append(shortener.shorten(aSuper));
+         sb.append(", ").append(shortener.shorten(aSuper));
```

Comment:
This is probably unintentional?

Reply:
Yes, it is. Thank you for pointing that out. I double-checked the indents in files, now there shouldn't be any stray whitespaces